starfish/experiment/builder/providers.py

```diff
@@ -25,6 +27,17 @@ def shape(self) -> Tuple[int, ...]:
         """
         raise NotImplementedError()
 
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        """Return the tile's coordinates in the global coordinate space..
+
+        Returns
+        -------
+        Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]
+            Maps from a coordinate type to its value or range.
```

Comment:
I'd make this more expressive. `Maps from a coordinate type (e.g. 'x', 'y', or 'z') to its value or range.`

Reply:
Fixed.